internal/evaluator/conftest_evaluator_test.go

```diff
@@ -1282,6 +1282,33 @@ func TestConftestEvaluatorIncludeExclude(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "warning for missing includes",
+			results: []Outcome{
+				{
+					Failures: []Result{
+						{Metadata: map[string]any{"code": "lunch.spam"}},
+						{Metadata: map[string]any{"code": "breakfast.spam", "term": []any{"bacon", "nocab"}}},
+					},
+				},
+			},
+			config: &ecc.EnterpriseContractPolicyConfiguration{Include: []string{"breakfast:eggs", "lunch"}},
```

Comment:
A nitpick:

Here "eggs" is a "term". Most of the time, we don't use a term in an include. (I just learned there is test coverage for doing that, but I don't think it's ever used in real life situation). The term is most often used for an exclude to do some fine-grained control of when the rule can be excluded.

It won't impact the test behavior, but I think it's more understandable if we make the missing include a rule, not a rule package and a term. E.g. something like this:

```suggestion
			config: &ecc.EnterpriseContractPolicyConfiguration{Include: []string{"breakfast.pancakes", "lunch"}},
```